CODEOWNERS

```
@@ -24,6 +24,8 @@ CODEOWNERS @MrAlias @MadVikingGod @pellared
 
 bridges/prometheus/                                                     @open-telemetry/go-approvers @dashpole
 
+config/                                                                 @open-telemetry/go-approvers @MadVikingGod
```

I can be added as well as I think I was reviewing all PRs so far.

```suggestion
config/                                                                 @open-telemetry/go-approvers @MadVikingGod @pellared
```